### ISSUE TITLE
corporation with chia-monitor? 

### ISSUE THREAD

**Issue by stefan-lange** [Chia-Network/pub-metrics-grafana#4]
This is a nice (if not the nicest) dashboard for chia
https://github.com/philippnormann/chia-monitor

A metrics mapper in your chia exporter or a talk to the author would be nice. There is no need to reinvent the wheel, but we can tune it 😃

Another go project is here, but it seems to be dead, but works with the old 1.2.11
https://github.com/retzkek/chia_exporter

**Comment by Starttoaster** [comment]
Hi! That looks like a really nice dashboard for a personal farm. The scope of the dashboards on this public Grafana instance is more blockchain-wide and not specific to one farm.

**Comment by stefan-lange** [comment]
Yep, it is. I created my own dashboard which works with the official chia-exporter from Chris.
https://github.com/stefan-lange/chia-farm-dashboard

ATM not all metrics are available. Maybe Chris has some time in the future to add at least a plot size metric. The request is here: https://github.com/Chia-Network/chia-exporter/issues/35

I'm no GO dev, but created the https://github.com/stefan-lange/chia-price-exporter with the help of the chia-exporter and gut feeling. It's working ;)

**Comment by Starttoaster** [comment]
Just read the code in your exporter :) Pretty cool! I'm personally not looking too closely at the price of XCH but I might set this up for my personal dashboards anyway. 

Also found it interesting to see a .gitlab-ci.yml file in a GitHub repo ;) 

Small nitpick, but in your gitlab CI yaml, I believe specifying an image globally was deprecated at some time in favor of the `default:image` keyword, which you could also probably leverage (the `default` keyword) for part of your yaml anchor. 😜

**Comment by Starttoaster** [comment]
Also, especially if this was your first time working in Go, great work!

**Comment by stefan-lange** [comment]
I'm in "team gitlab". Use it for all my stuff, at work and I like it. But the audience for such projects is on github. Repo mirroring does the trick for me.

Yep, that was the first time go.

Thx for the hint with the default image. I've been using Gitlab since 2017 and you never know enough :D
